src/renderless/CoreTable.vue

```diff
@@ -320,14 +321,65 @@ export default {
                 }
             });
         },
-        readRequest(method, exportMode = false, selection = false) {
+        fetchRow(dtRowId) {
```

Comment:
can't we reuse most part of fetch()?

Reply:
I tried to do it but the final code was a bit dirtier. because fetchRow() has something in the middle of the method and for refactoring this we should refactor methods to many tiny methods but it makes our class a little complicated(because it isn't SRP!)